apprentice-react-o-shop/src/services/userService.js

```diff
@@ -24,7 +26,12 @@ function logout() {
 }
 
 function register(user) {
-    //
+    const {username, password, lastName, firstName} = {...user}
```

Comment:
This was destructured here just to be put back together on line 31. Remove this line and put user into line 31.